functions/index.js

```diff
@@ -356,20 +391,20 @@ const createComment = (req, res) => {
         comment: req.body.comment,
 
     };
-    if (comment.comment === undefined) {
+    if (newComment.comment === undefined) {
         return res.status(400).json({ error: "Missing require parameter" });
     }
```

Comment:
The comments belong to the posts, so you need to receive a postId.

If we don't do this there's no way we can know which comment belongs to which post